ai/langchain4j/src/test/java/io/quarkus/ts/langchain4j/AbstractChatbotIT.java

```diff
@@ -0,0 +1,115 @@
+package io.quarkus.ts.langchain4j;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.WebSocket;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import org.awaitility.Awaitility;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.Protocol;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.services.URILike;
+
+@DisabledOnNative(reason = "https://issues.redhat.com/browse/QUARKUS-6774")
+public abstract class AbstractChatbotIT {
+    protected static final Logger LOG = Logger.getLogger(ChatbotIT.class);
```

Comment:
This is weird, `AbstractChatbotIT` is using logger of the `ChatbotIT`?

FYI - you can _optionally_ use logging in our FW, just do `Log.info(...)`

Reply:
Good catch, changed the logger